docs/sugar/04-configuration.md

```diff
@@ -0,0 +1,42 @@
+# Configuration
+
```

Comment:
Add an :::info::: block here:

Most users should use the `create-config` command to create the config file as that will ensure the generation of a valid config file and is easier to use for non-technical users.